### PR TITLE
WT-8161 Reduce verbosity of CMake Evergreen smoke

### PR DESCRIPTION
**Pull request by alisonfel**
Currently when ctest executes the smoke target, it does so with the verbosity option '-VV'. When parsing the logs for potential failures, this gets extremely noisy and hard to parse, since multiple parallel test outputs are interleaved with each other. 

To simplify this, remove the 'VV' option and replace it with 'output-on-failure'. This ensures all the output for a failing test will be logged, whilst successful test output will not be logged.

### CODE REVIEW

**Comment by jiechenbo** [comment]
LGTM! Makes sense to limit output.

**Review by jiechenbo — APPROVED** [review]

**Review by lukech — APPROVED** [review]
LGTM
